main/src/ui/conversation_content_view/quote_widget.vala

```diff
@@ -29,6 +29,10 @@ namespace Dino.Ui.Quote {
             if (content_item.type_ == MessageItem.TYPE) {
                 var message = ((MessageItem) content_item).message;
                 this.message = Dino.message_body_without_reply_fallback(message);
+
+		if (message.body.has_prefix("/me ")) {
+			this.message = @"<i><b>$(Markup.escape_text(display_name))</b> " + this.message.substring(4) + "</i>";
```

Comment:
Since the label now uses markup, you have to escape every string that you don't want to contain markup. That means, also the message. Otherwise, the message "/me <b>test</b>" would be displayed with a bold "test" instead of the actual message.